client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/useInteractionsForm.ts

```diff
@@ -15,114 +15,117 @@ import useStyles from './InteractionEventFormStyles';
 import {duplicateIdsErrorMsg} from '../../ContactQuestioning/ContactQuestioning';
 
 const useInteractionsForm = (props: useInteractionFormIncome): useInteractionFormOutcome => {
-    const classes = useStyles();
```

Comment:
Are these changes caused by the formatting being wrong and you fixing them?

Reply:
I'm using webstrom, there's a feature that if you press ctrl+alt+L it will format all of your file, so it's all automatic and unintended

Reply:
Fair enough